### DISCUSSION TITLE
Welcome to PyMoDAQ Discussions!

### DISCUSSION THREAD

**Discussion by seb5g** [PyMoDAQ/PyMoDAQ#41] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BAZILA** [comment]
Bonjour j'ai certaines question par rapport à l'utilisation du logiciel

**Reply by quantumm** [reply]
Hi!
If you have questions about pymodaq do not hesitate to open a discussion [here](https://github.com/PyMoDAQ/PyMoDAQ/discussions).
